### ISSUE TITLE
Make mapping for l1/l2 or r1/r2 configurable

### ISSUE THREAD

**Issue by Sepalod** [xerpi/ds4vita#32]
Would it be possible to make the mapping of l1 / l2 or r1 / r2 configurable? Via remoteplay  in some games there are special button mappings for the vita which switches buttons from the normal ds4 to another button on the vita. For example r2 is mapped on r1. So it would be useful if you could reconfigure this mapping in ds4vita if needed. Just telling the vita that I pressed r1 if I press r2 on the ds4, for example

**Comment by xerpi** [comment]
Yeah it would be nice to have a configurable button mapping file but I'm not sure how the file format should be so I accept suggestions.

**Comment by StarrHD** [comment]
this would be nice but might require specific game profiles like freedom wars has 4 different control configurations.
